Kickstarter-iOS/Views/Controllers/LiveVideoViewController.swift

```diff
@@ -147,7 +147,7 @@ public final class LiveVideoViewController: UIViewController {
     let player = AVPlayer(url: url)
     let controller = AVPlayerViewController()
     controller.player = player
-    controller.videoGravity = AVLayerVideoGravityResizeAspectFill
+    controller.videoGravity = AVLayerVideoGravity.resizeAspectFill.rawValue
```

Comment:
Why the `rawValue` here and not the enum type directly?

Reply:
Because `videoGravity` is a String, while `resizeAspectFill` is a AVLayerVideoGravity (that is a typedef for NSString). So we have to use rawValue here to match types and avoid the `Cannot assign value of type 'AVLayerVideoGravity' to type 'String'` error.